h/static/scripts/annotator/plugin/domtextmapper.coffee

```diff
@@ -1,11 +1,26 @@
+# Simple IE autodetect function
+# See for example https://stackoverflow.com/questions/19999388/jquery-check-if-user-is-using-ie/21712356#21712356
+isIE = ->
+  ua = window.navigator.userAgent
+  ua.indexOf("MSIE ") > 0 or      # for IE <=10
+    ua.indexOf('Trident/') > 0 or # for IE 11
+    ua.indexOf('Edge/') > 0       # for IE 12
+
 # Annotator plugin providing dom-text-mapper
 class Annotator.Plugin.DomTextMapper extends Annotator.Plugin
 
   pluginInit: ->
     if @options.skip
-      console.log "Not registering DOM-Text-Mapper."
+      console.log "Not registering DOM-Text-Mapper, since it's disabled."
       return
 
+    unless @options.force
```

Comment:
> what's options.force for?

It's there to disable auto-detection, and simply enable d-t-m for all users. (For whatever reason.)

Most importantly, it's a way to place the decision back to the hands of the integrator / deployer.